### PR TITLE
EQP updates

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) moves the location of the Marketplace EQP documentation and updates the content for the Early Access Program.

## Affected DevDocs pages

Many

whatsnew
Updated the [Marketplace EQP API](https://devdocs.magento.com/marketplace/eqp/v1/api.html) documentation for the Early Access Program.

### CODE REVIEW

File: marketplace/eqp/v1/filtering.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+group: marketplace-api
+title: Filtering
+---
+
+Some GET batch request endpoints support sorting, filtering, and pagination.
+
+Currently, the `files` endpoints support only pagination, while package endpoints support all three.
```
Comment (jeff-matthews) [inline marketplace/eqp/v1/filtering.md]:
```suggestion
Currently, the `files` endpoints support pagination only. The `package` endpoints support all three.
```

File: marketplace/eqp/v1/filtering.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+## Paginating batch requests
+
+The following GET request parameters are passed to request a subset of the results.
+
+|Field|Type|Default|Description|
+|-----|----|-------|-----------|
+|offset|int|0|The record in the list from which to begin. The list starts at 0.|
+|limit|int|20|Number of records to return, starting from `offset`. -1 will return all remaining records.|
+
+The result comes with a header `X-Total-Count`, which gives the number of total records.
```
Comment (jeff-matthews) [inline marketplace/eqp/v1/filtering.md]:
```suggestion
The result includes a `X-Total-Count` header, which provides the total number of records.
```

File: marketplace/eqp/v1/filtering.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+## Sorting batch requests
+
+If a field is listed as filterable, it is also sortable. To sort, pass in the `sort` GET parameter:
+
+|Field|Type|Description|
+|-----|----|-----------|
+|sort|string|Comma-separated list of field names to sort by. Fields may be prefixed with `-` to sort in descending order, or `+` for ascending.|
+
+For example, to sort all versions of all packages, M2 packages first, then M1,
```
Comment (jeff-matthews) [inline marketplace/eqp/v1/filtering.md]:
```suggestion
For example, to sort all versions of all packages (M2 packages first, then M1)
```

File: marketplace/eqp/v1/filtering.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+grouped alphabetically by name, with newest packages first:
```
Comment (jeff-matthews) [inline marketplace/eqp/v1/filtering.md]:
```suggestion
grouped alphabetically by name with the newest packages first:
```

File: marketplace/eqp/v1/filtering.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+```http
+GET /rest/v1/products/packages/?sort=-platform,+name,-version
+```
+
+## Filtering batch requests
+
+You can use multiple response fields as filters in batch GET requests.
+
+For example, to get a package with the submission_id "12345", one way is
+to call the single-object convenience endpoint, without using filters:
```
Comment (jeff-matthews) [inline marketplace/eqp/v1/filtering.md]:
```suggestion
to call the single-object convenience endpoint without using filters:
```

File: marketplace/eqp/v1/packages.md [hunk 6]
```diff
@@ -3,13 +3,16 @@ group: marketplace-api
 title: Packages
 ---
 
-Use this resource to initiate and manage all aspects of submitting a package to the [Magento Extension Quality Program (EQP)](https://docs.magento.com/marketplace/user_guide/sellers/extension-quality-program.html). You can provide all metadata associated with a package—both the technical and the marketing information—in a single step or in several steps using incremental updates.
+Use this resource to initiate and manage all aspects of submitting a package to the
+[Magento Extension Quality Program (EQP)](http://docs.magento.com/marketplace/user_guide/extensions/extension-quality-program.html).
+You can provide all metadata associated with a package, both the technical and the marketing information, in a single step, or in several steps, using incremental updates.
 
-*  **Technical information**—References to code artifacts, such as Magento 1 TGZ files or Composer-compliant Magento 2 ZIP files, version compatibility, and release notes.
+*  **Technical information** - References to code artifacts, such as Magento 1 TGZ files or Composer-compliant Magento 2 ZIP files, version compatibility, and release notes.
```
Comment (jeff-matthews) [inline marketplace/eqp/v1/packages.md]: The original em dash was correct (no spaces on either side).

File: marketplace/eqp/v1/packages.md [hunk 7]
```diff
@@ -3,13 +3,16 @@ group: marketplace-api
...
 
-*  **Marketing information**—Includes package descriptions, image assets for logos and galleries, pricing information, support and installation services offered, and various guides—user, installation, and reference in PDF.
+*  **Marketing information** - Includes package descriptions, image assets for logos and galleries, pricing information, support and installation services offered, and various guides (user, installation, and reference) in PDF.
```
Comment (jeff-matthews) [inline marketplace/eqp/v1/packages.md]: Same here

File: marketplace/eqp/v1/packages.md (hunk 6 above)
Comment (jeff-matthews) [inline marketplace/eqp/v1/packages.md]:
```suggestion
*  **Technical information**—References to code artifacts, such as Magento 1 TGZ files or Composer-compliant Magento 2 ZIP files, version compatibility, and release notes.
```

File: marketplace/eqp/v1/packages.md (hunk 7 above)
Comment (jeff-matthews) [inline marketplace/eqp/v1/packages.md]:
```suggestion
*  **Marketing information**—Includes package descriptions, image assets for logos and galleries, pricing information, support and installation services offered, and various guides (user, installation, and reference) in PDF.
```

File: marketplace/eqp/v1/packages.md [hunk 8]
```diff
@@ -379,15 +465,18 @@ PUT /rest/v1/products/packages/:item_id
...
-*  A single POST request with all required fields set. You must explicitly indicate that you are for technical and marketing review using the `action` parameter.
+*  A single POST request with all required fields set. You must explicitly indicate that you are submitting for technical and marketing review using the `action` parameter.
 *  A series of requests, typically in the following order:
    1. A single POST request with the minimum required fields set and `action` set to `draft` in either `technical`, `marketing`, or both. This request accepts the new package and saves it on the Developer Portal for further updates. It returns a unique `submission_id` for subsequent PUT operations.
    1. One or more PUT requests in which you configure the package parameters. In these requests, set `action` to `draft` in `technical`, `marketing`, or both.
    1. A final PUT request indicating submission for `technical`, `marketing`, or both.
 
-You can update one or more parameters in `draft` mode, but the API does not check for validation errors.
+You can update one or more parameters in `draft` mode.
+In this mode, the API checks only for basic type-validation issues.
```
Comment (jeff-matthews) [inline marketplace/eqp/v1/packages.md]:
```suggestion
In this mode, the API checks for basic type-validation issues only.
```

File: marketplace/eqp/v1/packages.md [hunk 9]
```diff
@@ -512,25 +603,33 @@ curl -X POST \
...
 
 *  The API returns a HTTP 200 batch response listing items in the same order as they were provided in the request.
-*  Each item contains a `code` and `message` indicating success or failure. Any non-200 code indicates an error. Refer to the message for more details.
-*  A unique `submission_id` is returned for each successful item, which must be used for any GET, PUT, or DELETE methods.
-*  Optionally, if a user-defined `item_id` was supplied during the POST, the response will echo back the same `item_id` for each item in the batch. The resource can be retrieved via GET using the `item_id`.
+*  Each item contains a `code` and `message` indicating success or failure. Any non-200 code indicates an error.
+   Refer to the message for more details.
+*  A unique `submission_id` is returned for each successful item, which must be used for any GET or PUT methods.
+*  Optionally, if a user-defined `item_id` was supplied during the POST,
+   the response will echo back the same `item_id` for each item in the batch.
```
Comment (jeff-matthews) [inline marketplace/eqp/v1/packages.md]:
```suggestion
   the response will return the same `item_id` for each item in the batch.
```

File: marketplace/eqp/v1/packages.md [hunk 10]
```diff
@@ -512,25 +603,33 @@ curl -X POST \
...
 
 *  The package is in draft mode for the technical or marketing review; or both.
-*  The package has been rejected in either the technical or marketing review; or both. You must fix these issues and re-submit the package.
+*  The package has been rejected in either the technical or marketing review; or both.
+   You must fix these issues and re-submit the package.
 *  The package has been released to the Magento Marketplace.
 *  The package was removed from the store by the developer and needs to be re-published.
 *  The package can be recalled while in the EQP pipeline.
-*  After a package has been released to the Magento Marketplace, you can update marketing information only. Changing marketing information causes the package to be placed in marketing review. The package continues to be live on the marketplace, and after the marketing approval, the updated fields will be published to the store.
+*  After a package has been released to the Magento Marketplace, you can update marketing information only.
```
Comment (jeff-matthews) [inline marketplace/eqp/v1/packages.md]:
```suggestion
*  After a package has been released on the Magento Marketplace, you can update marketing information only.
```

File: marketplace/eqp/v1/packages.md [hunk 11]
```diff
@@ -512,25 +603,33 @@ curl -X POST \
...
+   Changing marketing information causes the package to be placed in marketing review.
```
Comment (jeff-matthews) [inline marketplace/eqp/v1/packages.md]:
```suggestion
   Changes to marketing information require marketing review.
```

File: marketplace/eqp/v1/packages.md [hunk 12]
```diff
@@ -512,25 +603,33 @@ curl -X POST \
...
+   The package continues to be live on the marketplace, and after the marketing approval,
```
Comment (jeff-matthews) [inline marketplace/eqp/v1/packages.md]:
```suggestion
   The package continues to be live on the marketplace, and after approval,
```

File: marketplace/eqp/v1/packages.md [hunk 13]
```diff
@@ -540,23 +639,24 @@ The PUT method also allows for updating a single package with `submission_id` as
...
+     -d '{ "action" : { "overall" : "publish"} }' \
      https://developer-api.magento.com/rest/v1/products/packages/f4eacd72be
 ```
 
 The HTTP response code will indicate success or failure.
 
 #### Required parameters
 
-If the **action** parameter indicates `submit` value for technical, or marketing or both, the required parameters are listed below by their respective parallel EQP pipelines:
+If the **action** parameter gives a `submit` value for technical, marketing or both,
```
Comment (jeff-matthews) [inline marketplace/eqp/v1/packages.md]:
```suggestion
If the **action** parameter contains a `submit` value for technical, marketing or both,
```

File: marketplace/eqp/v1/users.md [hunk 14]
```diff
@@ -383,7 +389,8 @@ A 204 No Content HTTP response code indicates a successful update.
 
 ## Reports
 
-Use this API to retrieve reports owned by a specific user. Reports contain information about extensions sales, payout status, aggregate sales, refund data, and more.
+Use this API to retrieve reports owned by a specific user.
+Reports contain information about extensions sales, payout status, aggregate sales, refund data, and more.
```
Comment (jeff-matthews) [inline marketplace/eqp/v1/users.md]:
```suggestion
Reports contain information about extension sales, payout status, aggregate sales, refund data, and more.
```

File: marketplace/eqp/v1/users.md [hunk 15]
```diff
@@ -11,7 +11,8 @@ profile. You must create a new profile on the [Developer Portal](https://develop
 
 ## Profile
 
-You must use the `mage_id` associated with the client application in your developer account when making request to these endpoints. You can get this ID when obtaining a [session token]({{ page.baseurl }}/marketplace/eqp/auth.html#session-token).
+You must use the `mage_id` associated with the client application in your developer account when making
```
Comment (jeff-matthews) [inline marketplace/eqp/v1/users.md]: Why is there a line break here?

File: marketplace/eqp/v1/users.md [hunk 16]
```diff
@@ -169,25 +171,26 @@ The following example shows a request to update the personal profile bio field:
...
   }
 }
 ```
 
 The `action` field specifies which update operation to perform:
 
-*  `publish`—The default if not specified. Publishes the profile to the relevant [Marketplace Store Partners page](https://marketplace.magento.com/partners.html).
-*  `draft`—The update is saved on the Developer Portal, but not published.
+*  `publish` &mdash; The default if not specified. Publishes the profile to the relevant
+   [Marketplace Store Partners page](https://marketplace.magento.com/partners.html).
```
Comment (jeff-matthews) [inline marketplace/eqp/v1/users.md]: Why is there a new line here?

File: marketplace/eqp/v1/users.md [hunk 17]
```diff
@@ -169,25 +171,26 @@ The following example shows a request to update the personal profile bio field:
...
+*  `draft` &mdash; The update is saved on the Developer Portal, but not published.
```
Comment (jeff-matthews) [inline marketplace/eqp/v1/users.md]: Another new line?

**Comment by keharper** [comment]
running tests

**Pull request merged**